### PR TITLE
Adds ability to specify vertical offset for modal

### PR DESCRIPTION
**Pull request by aszenz**

### CODE REVIEW

File: src/Ant/Modal.elm [hunk 1]
```diff
@@ -273,6 +279,16 @@ withMask toggle (Modal opts contents) =
     Modal { opts | showMask = toggle } contents
 
 
+{-| Specify a percentage value for positioning the modal vertically with respect to the documents height.
+
+By default this value is set to `15`.
+
+-}
+withVerticalOffset : Float -> Modal msg -> Modal msg
```
Comment (supermacro) [inline src/Ant/Modal.elm:287]: I think this function should allow for either percentage or pixel-based offsets. For example, in the case of the parlez-vous embed, the document height changes based on the number of comments rendered. So having a percentage based offset would mean that the modal would render at unpredictable offsets on various comment tree configurations.

File: src/Ant/Modal.elm [hunk 2]
```diff
@@ -356,13 +372,10 @@ closeIcon opts =
...
     in
-    toUnstyled styledHtml
-
-
-
--- TODO: add type anotation
+    styledHtml
 
 
+makeBorder : (Px -> BorderStyle (TextDecorationStyle {}) -> Color -> a) -> a
```
Comment (supermacro) [inline src/Ant/Modal.elm:378]: Thanks 🙂

File: src/Ant/Modal.elm [hunk 3]
```diff
@@ -487,12 +500,19 @@ toHtml isVisible (Modal opts contents) =
                     attr "2-row"
 
         modalHtml =
-            H.div [ class modalContainerClass, layoutInfoAttr ]
+            S.div
+                [ A.class modalContainerClass
+                , layoutInfoAttr
+                , css
+                    [ top (pct opts.verticalOffset)
```
Comment (supermacro) [inline src/Ant/Modal.elm:507]: See related comment above. Would be nice to have a union type to allow for either percent or pixel offsets.

**Review by supermacro — CHANGES_REQUESTED** [review]
Looks good, just requesting that we also allow for pixel-based offset.

Secondly, I don't know why the CircleCI jobs aren't running 😞  .. but if you can, please make sure that your files are formatted by running:

```
> ./scripts/format-elm-files.sh
```

File: src/Ant/Modal.elm (hunk 2 above)
Comment (aszenz) [reply src/Ant/Modal.elm:390]: welcome, but this one the elm lsp auto inferred for me

File: src/Ant/Modal.elm (hunk 1 above)
Comment (aszenz) [reply src/Ant/Modal.elm]: good idea, i have made two functions for this to keep the api simpler and avoid having to expose the union type

**Comment by aszenz** [comment]
> Looks good, just requesting that we also allow for pixel-based offset.
> 
> Secondly, I don't know why the CircleCI jobs aren't running  .. but if you can, please make sure that your files are formatted by running:
> 
> ```
> > ./scripts/format-elm-files.sh
> ```

ran the format script, as for circle ci maybe there doesn't seem to be any logs to check

**Review by supermacro — APPROVED** [review]

**Pull request merged**